Source: @pullmoll
Upstream: no
Reason: Not needed. For glibc this has just #include <signal.h> and for musl it prints a warning.

--- include/qemu/osdep.h	2020-12-08 17:59:44.000000000 +0100
+++ include/qemu/osdep.h	2020-12-12 18:41:47.709685865 +0100
@@ -104,10 +104,6 @@
 #include <setjmp.h>
 #include <signal.h>
 
-#ifdef HAVE_SYS_SIGNAL_H
-#include <sys/signal.h>
-#endif
-
 #ifndef _WIN32
 #include <sys/wait.h>
 #else
